.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.html"
+      - "**.css"
+      - "**.js"
+      - "**.md"
+      - "**.tsx"
+      - "**.json"
```

Comment:
```suggestion
```
[Matt's comment makes sense, I guess we could actually trust ourselves...](https://github.com/connectionsmuseum/public-web/pull/6/files/e1e2db25ccacbc66717659e666ea3ae3f3096562#r2404793045)